opentripplanner-geocoder/src/main/resources/org/opentripplanner/geocoder/application-context.xml

```diff
@@ -174,21 +202,31 @@
 				<entry key="(RZE) Zeta Hall" value="28.06670 -82.41264" />
 
 				<entry key="(SCA) Science Center" value="28.0605 -82.4158" />
+				<entry key="(SCD) Softball Stadium" value="28.05796 -82.40445" />
+				<entry key="(SEC) SE Chiller Plant" value="28.05923 -82.40744" />
+				<entry key="(SHR) Shriners Hospital for Children" value="28.06163 -82.42278" />
 				<entry key="(SHS) Student Health Services Building" value="28.0636 -82.4119" />
 				<entry key="(SOC) Social Science Building" value="28.0615 -82.4092" />
-				<entry key="(SPS) Corbett Soccer Park Stadium" value="28.06007 -82.40383" />
+				<entry key="(SPS) Soccer Stadium" value="28.06007 -82.40383" />
+				<entry key="(SRB) Stabile Research Building" value="28.06565 -82.42144" />
+				<entry key="(STA) Stadium" value="28.06302 -82.40670" />
+				<entry key="(SUN) Sun Dome" value="28.05917 -82.40654" />
 				<entry key="(SVC) Student Services Building" value="28.0625 -82.4124" />
 
 				<entry key="(TAR) Theatre Centre" value="28.0640 -82.4145" />
 				<entry key="(TAT) Theatre Auditorium - Theatre I" value="28.0636 -82.4145" />
 				<entry key="(THR) Theatre II" value="28.0636 -82.4149" />
 				<entry key="(TVB) WUSF Television Station" value="28.0623 -82.4118" />
 
+				<entry key="(UDI) University Diagnostic Institute (UDI)" value="28.05721 -82.42123  " />

```

Comment:
I tried deploying geocoder changes in tomcat locally and test them, but when application requests the geocoder results locally, the request is rejected because of the connection is not private. I tried adding keystore file in apache tomcat, but it stills rejects the request saying certificate error and so. Probably, I can only test after committing these changes.

I also think that extra whitespace is probably fine. I will recheck it once the changes are committed.

latlngs are taken from USF maps. I took the latlng values by dropping flag somewhere nearly center of the building. I can test whether latlngs generated are in sync with the nominatim values on how the routes are generated once the changes are committed.